src/main/java/core/basesyntax/serviceimpl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> readFromFile(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t find the file " + fileName, e);
+        }
+    }

```

Comment:
The `FileReaderImpl` class correctly implements the `FileReader` interface and uses try-with-resources to handle file reading which is good. However, it's recommended to avoid hardcoding file paths or file names inside the service classes. It would be better if the file path was injected into the `FileReaderImpl` class, for example, through a constructor or a setter method. This would make the class more flexible and easier to reuse in different contexts.